cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -551,4 +552,24 @@ public static String getJavaSystemPropertiesToString(List<SystemProperty> javaSy
         }
         return String.join(" ", javaSystemPropertiesList);
     }
+
+    public static void removeEmptyValuesFromTolerations(List<Toleration> tolerations) {
```

Comment:
Javadocs please ... plus should this be really private? It does not seem to be used from anywhere else than `tolerations(...)`.